src/components/containers/login.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { bindActionCreators } from "redux";
+import * as AuthenticationActions from "../../actions/authenticationActions";
+import LoginModal from "../views/loginModal";
+
+/* eslint-disable react/no-unused-state */
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      emailError: "",
+      disableLoginButton: true
+    };
+
+    this.onHandleChange = this.onHandleChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  onHandleChange = (event) => {
```

Comment:
since you are binding this method stop using an arrow function and convert it to class methods.

Reply:
working on the changes